web/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -35,6 +35,12 @@
                             {{/if}}
                         </span>
                     </div>
+                    <div class="unsubscribed_streams_tab_empty_text">
```

Comment:
Maybe `notsubscribed_streams_tab_empty_text` is more appropriate.

Reply:
Yup, changed it